src/doc/src/reference/environment-variables.md

```diff
@@ -21,15 +21,17 @@ system:
 * `RUSTC` — Instead of running `rustc`, Cargo will execute this specified
   compiler instead. See [`build.rustc`] to set via config.
 * `RUSTC_WRAPPER` — Instead of simply running `rustc`, Cargo will execute this
-  specified wrapper instead, passing as its command-line arguments the rustc
-  invocation, with the first argument being `rustc`. Useful to set up a build
-  cache tool such as `sccache`. See [`build.rustc-wrapper`] to set via config.
-* `RUSTC_WORKSPACE_WRAPPER` — Instead of simply running `rustc`, Cargo will
-  execute this specified wrapper instead for workspace members only, passing
+  specified wrapper, passing as its command-line arguments the rustc
+  invocation, with the first argument being the path to the actual rustc.
+  Useful to set up a build cache tool such as `sccache`. See
+  [`build.rustc-wrapper`] to set via config. Setting this to the empty string
+  overwrites the config and resets cargo to not use a wrapper.
```

Comment:
Workspace wrapper can also be reset with the same trick. However, this seems to be an undocumented behaviour, [attributed to these lines](https://github.com/rust-lang/cargo/blob/ba5b1920c4f0797bf30b30eb152265586c7f661e/src/cargo/util/config/mod.rs#L1470-L1493).

IMHO, if users start to rely on an undocumented but reasonable behaivour, we should treat it as a feature and document it well, like what you just did.

Reply:
I feel like the reset is reasonable to document. Could you mention it for rustc-workspace-wrapper as well?

Reply:
Oh. If this is going to be documented, adding some tests to prevent regression would be better. Sorry for the reply bombing.

Reply:
> Could you mention it for rustc-workspace-wrapper as well?

Done.

However I won't have the time to add tests, sorry.

Reply:
I'm not sure if this is what you were referring to, but I opened #10899 to update the test to make this behavior explicitly tested.